yarn.lock

```diff
@@ -1717,6 +1717,11 @@ array-differ@^1.0.0:
   resolved "https://registry.yarnpkg.com/array-differ/-/array-differ-1.0.0.tgz#eff52e3758249d33be402b8bb8e564bb2b5d4031"
   integrity sha1-7/UuN1gknTO+QCuLuOVkuytdQDE=
 
+array-differ@^2.0.3:
```

Comment:
Should this file be `.gitignore`d?

Reply:
Nope, it helps ensure the exact packages are installed for two different people.